sycl/test/extensions/usm_pointers_aliases.cpp

```diff
@@ -1,12 +1,12 @@
-// RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify %s -o %t.out
-// expected-no-diagnostics
+// RUN: %clangxx %fsycl-host-only -fsyntax-only -Xclang -verify -Xclang -verify-ignore-unexpected=note %s
```

Comment:
I assume that deprecation `decorated::legacy` deprecation warnings are not the main thing we want to test here. Therefore, we can consider adding `-Wno-deprecated` and leave the test as-is, i.e. expecting no diagnostics

Reply:
Thanks @AlexeySachkov , good point.